integ/scripts/bash/rfdk-integ-e2e.sh

```diff
@@ -90,11 +91,29 @@ for COMPONENT in **/cdk.json; do
     export ${COMPONENT_NAME}_START_TIME=$SECONDS
     if [[ "$COMPONENT_NAME" != _* ]]; then
         # Excecute the e2e test in the component's scripts directory
-        cd "$INTEG_ROOT/$COMPONENT_ROOT" && ../common/scripts/bash/component_e2e.sh "$COMPONENT_NAME" || cleanup_on_failure
+        cd "$INTEG_ROOT/$COMPONENT_ROOT"
+        ../common/scripts/bash/component_e2e.sh "$COMPONENT_NAME" || cleanup_on_failure &
```

Comment:
Have you tested this change with a failing integration test? ex: Make a modification to one of the jest tests that would cause a failure, and make sure that the behavior is as expected.

Reply:
I have tested it and I still can see errors in console like it was before:
```
 FAIL  components/deadline/deadline_02_renderQueue/test/deadline_02_renderQueue.test.ts (53.564 s)
  ● Deadline RenderQueue tests (HTTP mode) › deadlinecommand tests › RQ-1-4: Farm can accept sample Deadline job
    expect(received).toEqual(expected) // deep equality

    Expected: 10
    Received: 0

      228 |       return awaitSsmCommand(bastionId, params).then( response => {
      229 |         var responseCode = response.responseCode;
    > 230 |         expect(responseCode).toEqual(10);
          |                              ^
      231 |       });
      232 |     });
      233 |   });

      at components/deadline/deadline_02_renderQueue/test/deadline_02_renderQueue.test.ts:230:30
```
And result as expected:
```
Results for test component deadline_02_renderQueue:
  -Tests ran: 8
  -Tests passed: 6
  -Tests failed: 2
  -Deploy runtime:     24m 16s
  -Test suite runtime: 0m 55s
  -Cleanup runtime:    1m 43s
```